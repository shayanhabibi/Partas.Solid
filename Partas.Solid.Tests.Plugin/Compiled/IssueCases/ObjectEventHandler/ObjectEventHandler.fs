module Partas.Solid.Tests.Plugin.Compiled.IssueCases.ObjectEventHandler.ObjectEventHandler

open Partas.Solid
open Fable.Core

[<SolidComponent>]
let test () =
    div().on("event", OnHandler((fun e -> JS.console.log(e)), passive = true))

