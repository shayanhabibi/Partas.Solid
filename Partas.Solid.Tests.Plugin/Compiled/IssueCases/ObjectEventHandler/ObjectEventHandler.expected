import { some } from "../../fable_modules/fable-library-js.5.0.0-alpha.14/Option.js";

export function test() {
    return <div on:event={{
            handleEvent: (e) => {
                console.log(some(e));
            },
            passive: true,
        }} />;
}

